src/main/java/org/nguh/nguhcraft/block/NguhBlocks.kt

```diff
@@ -235,6 +227,293 @@ object NguhBlocks {
             .mapColor(MapColor.STONE_GRAY)
     )
 
+    // Cinnabar blocks
+    val CINNABAR = Register(
+        "cinnabar",
+        ::Block,
+        AbstractBlock.Settings.copy(Blocks.TUFF)
+            .mapColor(MapColor.DARK_RED)
+    )
+
+    val CINNABAR_SLAB = RegisterVariant(CINNABAR, "slab", ::SlabBlock)
+    val CINNABAR_STAIRS = RegisterStairs(CINNABAR)
+
+    val POLISHED_CINNABAR = Register(
+        "polished_cinnabar",
+        ::Block,
+        AbstractBlock.Settings.copy(Blocks.STONE)
+            .mapColor(MapColor.DARK_RED)
+    )
+
+    val POLISHED_CINNABAR_SLAB = Register(
+        "polished_cinnabar_slab",
+        ::SlabBlock,
+        AbstractBlock.Settings.copy(Blocks.TUFF)
+            .mapColor(MapColor.DARK_RED)
+    )
+
+    val POLISHED_CINNABAR_STAIRS = Register(
+        "polished_cinnabar_stairs",
+        ::StairsBlock,
+        AbstractBlock.Settings.copy(Blocks.TUFF)
+            .mapColor(MapColor.DARK_RED)
+    )
+
+    val POLISHED_CINNABAR_WALL = Register(
+        "polished_cinnabar_wall",
+        ::WallBlock,
+        AbstractBlock.Settings.copy(Blocks.TUFF)
+            .mapColor(MapColor.DARK_RED)
+    )
+
+    val POLISHED_CINNABAR_BUTTON = Register(
+        "polished_cinnabar_button",
+        ::Block,
+        AbstractBlock.Settings.copy(Blocks.CALCITE)
+            .mapColor(MapColor.TERRACOTTA_WHITE)
+    )
+
+    val POLISHED_CINNABAR_PRESSURE_PLATE = Register(
+        "polished_cinnabar_pressure_plate",
+        ::Block,
+        AbstractBlock.Settings.copy(Blocks.CALCITE)
+            .mapColor(MapColor.TERRACOTTA_WHITE)
+    )
```

Comment:
I’m going to nak all the buttons and pressure plates, so please remove those; those blocks are too involved.